src/Footer.js

```diff
@@ -0,0 +1,8 @@
+var React = require('react');
+
```

Comment:
check out this pattern for rendering presentational components: 

https://github.com/sadams/todo-redux-react-webpack/blob/master/app/js/components/Footer.jsx#L55